src/Storage/AxolotlStore/TSStorageManager+Calling.m

```diff
@@ -0,0 +1,26 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "TSStorageManager+Calling.h"
+
+#define TSStorageManagerCallKitIdToPhoneNumberCollection @"TSStorageManagerCallKitIdToPhoneNumberCollection"
```

Comment:
Done. Yup.  My policy is to try to clean up code as I touch it 90% of the time. This was a 10% exception since I was rushing to get the release ready before I go on vacation.